package.json

```diff
@@ -57,13 +53,13 @@
   "dependencies": {},
   "devDependencies": {
     "benchmark": "^2.1.0",
-    "browserify": "^13.0.0",
-    "browserify-istanbul": "^1.0.0",
-    "coveralls": "2.11.9",
-    "eslint": "^2.9.0",
-    "eslint-config-strict": "^8.5.0",
-    "eslint-plugin-filenames": "^0.2.0",
-    "ghooks": "^1.0.1",
+    "browserify": "^14.3.0",
```

Comment:
I think that we won't need to reference Buffer anytime soon given the purpose of this library.
If we ever need to we can always go back to the old version.

I agree with this change. Also, thanks for explaining, it's always good to have this kind of comment available on GitHub if we ever need to see the reason why we're doing this.